main/RFXNames.h

```diff
@@ -222,6 +222,11 @@ enum _eHardwareTypes {
 	HTYPE_AirconWithMe,         //123
 	HTYPE_TeleinfoMeterTCP,		//124
 	HTYPE_MQTTAutoDiscovery,	//125
+	HTYPE_RESERVED_FOR_OTHERS3,	//126
```

Comment:
Could you please not use 'HTYPE_RESERVED_FOR_OTHERS' here, but just your new hardware type ?
You can change the hardware type in your database